packages/env-spec-parser/src/simple-resolver.ts

```diff
@@ -28,6 +28,16 @@ export function simpleResolver(
         } else {
           throw new Error('Invalid `ref` args');
         }
+      } else if (valOrFn.name === 'replace') {
```

Comment:
We don't actually need this defined here at all. This is just a very simple implementation of a few functions which are used to test the parser itself.